authn_k8s_scale_demo/4_deploy.sh

```diff
@@ -13,7 +17,7 @@ kubectl create configmap webapp \
   --from-literal=ssl_certificate="$ssl_certificate"
 
 export CLIENT_API_KEY=$(conjur host rotate_api_key -h conjur/authn-k8s/minikube/default/client)
-echo Client API key: $CLIENT_API_KEY
+echo Environment token: $CLIENT_API_KEY
 
 # save client API key as k8s secret
 kubectl delete secret conjur-client-api-key
```

Comment:
`kubectl delete --ignore-not-found=true...` to avoid errors `Error from server (NotFound): secrets "conjur-client-api-key" not found`